docs/src/guides/gatsby/deploy/configure.md

```diff
@@ -7,17 +7,15 @@ description: |
     Review the basics of what makes up a Platform.sh project, including its three principle configuration files and how to define them for Gatsby.
 ---
 
-{{< guides/config-desc name="Gatsby" >}}
+{{< guides/config-desc name="Gatsby" noService=true >}}
 
 ## Requests configuration: `routes.yaml`
 
 {{< guides/config-routes template="gatsby" name="Gatsby" >}}
 
 ## Service configuration: `services.yaml`
 
-The `services.yaml` file lists the pre-packaged services you need for your application to run. You pick the major version of the service, and Platform.sh updates the patch version periodically so that you always get the newest version when you deploy.
-
-Gatsby does not require services to deploy, so you can leave the `services.yaml` file empty for now. You can add [other services](/configuration/services/_index.md) if desired, such as [Solr](/configuration/services/solr.md) or [Elasticsearch](/configuration/services/elasticsearch.md). You will need to configure Gatsby to use those services as well once the service is enabled.
+{{% guides/config-service name=Gatsby noService=true /%}}
```

Comment:
It's probably missing the parenthesis to work properly. See comment in `docs/layouts/shortcodes/guides/config-service.md`

```suggestion
{{% guides/config-service name="Gatsby" noService=true /%}}
```

Reply:
The quotation marks aren't necessary. You can tell because `Gatsby` appears in the second paragraph in the [deployed page](https://pr-2215-kyjvqka-652soceglkw4u.eu-3.platformsh.site/guides/gatsby/deploy/configure.html#service-configuration-servicesyaml)